lib/test_base.rb

```diff
@@ -60,43 +60,37 @@ module TestBase
   DRUBY_REMOTE_PORT = 27183
 
   # absolute path to the systemtests/tests/ directory as specified by RUBYLIB
-  TESTDIR = $:.find {|path| path =~ /systemtests.*\/tests/}
+  OLDTESTDIR = $:.find {|path| path =~ /systemtests.*\/tests/}
+  if OLDTESTDIR == nil
+    OLDTESTDIR = ""
```

Comment:
This is trying to change a constant. Changing the code to

OLDTESTDIR = $:.find {|path| path =~ /systemtests.*\/tests/} || ""

would eliminate the issue.  A better approach is to add a function to EnvironmentBase that returns the string.